src/persistence/settings.cpp

```diff
@@ -1175,6 +1177,19 @@ void Settings::setCamVideoRes(QSize newValue)
     camVideoRes = newValue;
 }
 
+unsigned short Settings::getCamVideoFPS() const
+{
+    QMutexLocker locker{&bigLock};
```

Comment:
Settings class works as low priority thread, it was designed like this by tux3. So if this is a confirmed performance problem, you should open issue for that or fix it.